mmdet/models/backbones/res2net.py

```diff
@@ -313,3 +317,34 @@ def make_res_layer(self, **kwargs):
             base_width=self.base_width,
             base_channels=self.base_channels,
             **kwargs)
+
+    def init_weights(self, pretrained=None):
+        """Initialize the weights in backbone.
+
+        Args:
+            pretrained (str, optional): Path to pre-trained weights.
+                Defaults to None.
+        """
+        if isinstance(pretrained, str):
+            logger = get_root_logger()
+            load_checkpoint(self, pretrained, strict=False, logger=logger)
+        elif pretrained is None:
+            for m in self.modules():
+                if isinstance(m, nn.Conv2d):
+                    kaiming_init(m)
+                elif isinstance(m, (_BatchNorm, nn.GroupNorm)):
+                    constant_init(m, 1)
+
+            if self.dcn is not None:
+                for m in self.modules():
+                    if isinstance(m, Bottle2neck):
+                        for n in m.convs:
```

Comment:
We may add comments here.
Like `dcn in Res2Net bottleneck is in ModuleList`.

Reply:
> We may add comments here.
> Like `dcn in Res2Net bottleneck is in ModuleList`.

ok, i will add comments